### PR TITLE
Aden 2187 ads tests update

### PR DESCRIPTION
**Pull request by gabrys**

### CODE REVIEW

File: src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/helpers/AdsComparison.java [hunk 1]
```diff
@@ -86,10 +86,13 @@ public boolean isAdVisible(final WebElement element, final String selector,
...
           BufferedImage adImg = shooter.takeScreenshot(element, driver);
           PageObjectLogging.log("ScreenshotsComparison", "Ad image in " + selector, true);
-          Triple rgb = ImageHelper.getRgbVariance(adImg);
-          PageObjectLogging.log("R variance", String.valueOf(rgb.getLeft()), true);
-          PageObjectLogging.log("G variance", String.valueOf(rgb.getMiddle()), true);
-          PageObjectLogging.log("B variance", String.valueOf(rgb.getRight()), true);
+//          Triple rgb = ImageHelper.getRgbVariance(adImg);
+//          PageObjectLogging.log("R variance", String.valueOf(rgb.getLeft()), true);
+//          PageObjectLogging.log("G variance", String.valueOf(rgb.getMiddle()), true);
+//          PageObjectLogging.log("B variance", String.valueOf(rgb.getRight()), true);
```
Comment (nandy-andy) [inline src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/helpers/AdsComparison.java]: Why we keep these lines if we comment them out? :japanese_ogre:

File: src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/mobile/MobileAdsBaseObject.java [hunk 2]
```diff
@@ -51,19 +53,25 @@ protected void setWindowSizeAndroid() {
...
-            return;
-        }
-        waitForSlotExpanded(presentLeaderboard);
+
+        removeElementIfPresent(SMART_BANNER_SELECTOR);
+
         if (!adsComparison.isAdVisible(presentLeaderboard, presentLeaderboardSelector, driver)) {
+
+            if (checkIfElementOnPage(CELTRA_MASK_SELECTOR)) {
+                PageObjectLogging.logWarning("Special ad", "Celtra");
```
Comment (nandy-andy) [inline src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/mobile/MobileAdsBaseObject.java]: What do you think about logging also line-item-it and creative-id here? Would it be useful?

File: src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/AdsBaseObject.java [hunk 3]
```diff
@@ -607,19 +607,21 @@ public AdsBaseObject refresh(int times) {
...
 
-    public AdsBaseObject waitPageLoaded() {
-        waitOnReadyEvent();
+    public AdsBaseObject waitForPageLoaded() {
+        waitForJavaScriptTruthy("document.readyState === 'complete'");
         return this;
     }
 
-    private void waitOnReadyEvent() {
+    private void waitForJavaScriptTruthy(final String script) {
```
Comment (drets) [inline src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/AdsBaseObject.java]: :+1:

File: src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/helpers/AdsComparison.java (hunk 1 above)
Comment (drets) [inline src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/helpers/AdsComparison.java]: +1 for removing it and `getRgbVariance` method

File: src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/mobile/MobileAdsBaseObject.java (hunk 2 above)
Comment (gabrys) [inline src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/mobile/MobileAdsBaseObject.java]: It's already done before.

**Comment by drets** [comment]
LGTM

**Pull request merged**